consensus/state.go

```diff
@@ -1830,9 +1830,11 @@ func (cs *State) recordMetrics(height int64, block *types.Block) {
 	if height > 1 {
 		lastBlockMeta := cs.blockStore.LoadBlockMeta(height - 1)
 		if lastBlockMeta != nil {
-			cs.metrics.BlockIntervalSeconds.Observe(
-				block.Time.Sub(lastBlockMeta.Header.Time).Seconds(),
-			)
+			elapsedTime := block.Time.Sub(lastBlockMeta.Header.Time).Seconds()
+			cs.metrics.BlockIntervalSeconds.Observe(elapsedTime)
+			cs.metrics.BlockTimeSeconds.With("block_height",
+				fmt.Sprintf("%d", block.Height)).Set(elapsedTime)
```

Comment:
[not blocking, just a comment]
do we even need the histogram version of blocktime after this? could be just create a query that measured the same information and generated a histogram instead of recording the same information twice?

Reply:
It seems doable using recording rules or by getting a bit creative with PromQL queries. But  both those options might seem a tad trickier compared to just using the histogram metric type. Nevertheless, it's worth noting that a histogram doesn't really consume up much space – it's constant- just 14 counters in total, spread across 12 buckets, plus a sum and a count. So, keeping it around might not be a bad idea due to its low overhead.